index.js

```diff
@@ -20,7 +20,8 @@ app.get('/auth',(req,res) => {
         qs.stringify({
             response_type: 'code',
             client_id: process.env.CLIENT_ID,
-            redirect_uri: `${process.env.SERVER_URL}/redirect`
+            redirect_uri: `${process.env.SERVER_URL}/redirect`,
+            scope: `${process.env.SCOPES}`
```

Comment:
As this is an optional config value, it would be good if this was implemented as optional for this project too.

Something like... (this is untested 😃)
```js
let authConfig = {
    response_type: 'code',
    client_id: process.env.CLIENT_ID,
    redirect_uri: `${process.env.SERVER_URL}/redirect`
}
if (process.env.SCOPES) {
  authConfig.scope = process.env.SCOPES;
}
res.redirect('https://accounts.spotify.com/authorize?' + qs.stringify(authConfig));
```